app/posts/find-teacher-training/2022-02-11-updating-international-candidates-section.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Updating the ‘International candidates’ section
+description: Why we have updated and renamed this section
+date: 2022-02-11
+screenshots:
+  items:
+    - text: International candidates - sponsorship may be available
+      src: find-sponsorship-available.png
+    - text: International candidates - sponsorship not available
+      src: find-sponsorship-not-available.png
+related:
+  items:
+    - text: GitHub pull request making the change
+      href: https://github.com/DFE-Digital/find-teacher-training/pull/1138
+    - text: Design history entry for previous change
+      href: /find-teacher-training/new-international-students-section/
+---
+
+In August last year, we [added a new ‘International students’ section](/find-teacher-training/new-international-students-section/) as a place where we could explain things that people applying from outside the UK need to know, including whether the course offered visa sponsorship or not.
+
+We have renamed this section ‘International candidates’ for more consistency with the ‘candidate’ term we use elsewhere, and because users are not students at the point of applying, and may also apply for a salaried course where the term ‘student’ is less applicable.
+
+We have also updated the content to:
+
+* explicitly explain that candidates already have the right to work and study if they have ‘indefinite leave to remain’
+* mention other visa types which do not need sponsorship, such as Graduate and Family visas
+* link to a guidance page on GOV.UK titled [Train to teach in England if you’re a non-UK citizen](https://www.gov.uk/government/publications/train-to-teach-in-england-non-uk-applicants/train-to-teach-in-england-if-youre-a-non-uk-citizen) which has more information.
+
+
+Where the information from Publish indicates that the provider can sponsor the relevant visa for the course (Student or Skilled worker), we have made the guidance less certain:
+
+> Before you apply for this course, contact us to check [visa type] visa sponsorship is available.
+
+This is because visa sponsorship availability is more complicated than we previously believed, and depends upon either who the accrediting body is (for Student visas) or who the employer is (for Skilled worker visas), and therefore the information as collected through Publish may not be accurate.
```

Comment:
I'm not sure this bit is clear. 'Depends on' in what sense?

Reply:
Ah. So as I understand it, for a School Direct course, the Student visa would be sponsored by the accrediting body (eg a University), so it’s whether they have the sponsorship licence that counts, rather than the course provider (which is what our current logic assumes).

For salaried courses, it might be that a course is offered by a partnership of schools and the employer will be one of the schools that the candidate is placed with. Some of those schools might have a visa sponsorship licence but others might not - so it depends upon that rather than the provider (the partnership) which is advertising the overall course.

The gist of all this is that with the current data from Publish, we don't always have enough information to be able to guarantee that visa sponsorship is available. 😩

Reply:
haha wish I hadn't asked!! :)